docs/src/main/paradox/scala/http/server-side/http2.md

```diff
@@ -0,0 +1,83 @@
+# HTTP/2 (Preview)
+
+HTTP/2 support in akka-http is currently available as a preview.
```

Comment:
Maybe prefix with "Server-side", so it's clear from the beginning that this is only supported for the server.